### ISSUE TITLE
[CAIA A11y Research - PILOT]: MHV Medications (May 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#83156]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATES:**  
>     - **Desktop**: 5/20/2024 9:30-11 a.m. ET
>     - **Mobile**: `TENTATIVE MAY NOT HAPPEN` 5/21 or 5/22
> - **TYPES OF ASSISTIVE TECH:** Screen Readers and TTS (devices and OS needed)

# CAIA A11y Support
This is a child ticket of the research ticket #82618 - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @sara-amanda (Mon. avail.) - `SECONDARY for Pilot 1`
- @coforma-jamie (Mon. avail.) - `PRIMARY for Pilot 1`
- @EvanAtCoforma (Tues/Wed avail.) - `PILOT 2 IS TENTATIVE`
- @SarahKay8 (Tues/Wed avail.) `PILOT 2 IS TENTATIVE`

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name:Medications on VA.gov
- OCTO-DE product owner: Kay Lawyer
- Product manager: Kaitlin Fink
- Designer: Lexi Wunder
- FE engineer:
- Product/team Slack channel: [#mhv-medications-rx](https://dsva.slack.com/archives/C04PRFEJQTY)

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session

@AnneCostello **shared the prototype link so we can see what the environment will look like. MHV Medications will be using staging for the actual sessions.**

- **CAIA A11Y Pilot Date:** `TBD`
- [Prototype or mockups](https://www.figma.com/proto/UGHasFKuKENvKz8AgSmxtI/Medications?node-id=4942-23559&t=5AGkqQpTPeDlSNuA-1&scaling=scale-down&page-id=4942%3A23558&starting-point-node-id=4942%3A23559)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/research/2024-05-medications-usability-testing-round3-AT/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/research/2024-05-medications-usability-testing-round3-AT/conversation-guide-sr-vc.md)
- **Staging Link:** `TBD`

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [X] **Schedule CAIA a11y:** 5/20/2024 9:30 AM ET (PILOT 1)
- [x] Contact CAIA for pilot (Anne on 5/15/2024)
- [x] Create pilot ticket (Sara on 5/15/2024)
- [ ] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**  5/20/2024 at 9:30 a.m. ET
- [x] **Update CAIA A11y Capacity Calendar:** Add HOLD invite for pilot (adding hold 5/16 for 5/20)
- [X] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [X] **Review research** [conversation guide](link needed) and provide feedback in comments.
- [X] [Request AT device and tech to be used in sessions](LINK)
- [ ] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [ ] **Review prototype** in `Location` - `STATUS OF PROTOTYPE`
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. Folder for Team in GH Deliverables for CAIA A11Ys.
```
